### ISSUE TITLE
CODESPACES: Resource Not Accessible by Integration when creating a PR

### ISSUE THREAD

**Issue by burkeholland** [microsoft/vscode-pull-request-github#2450]
- Extension version: 2021.2.1982
- Codespaces: Production(e035230) 
- OS: Windows 10

## Steps to Reproduce:

1. Fork a public repo
2. Add the parent as an upstream
3. Create a branch
4. Make a change
5. Stage & Commit
6. Push branch
7. Select "yes" when asked if you would like to create a PR
8. Select the upstream from the dropdown
9. Take the default title
10. Click "Create"
1

### What Should Happen

A PR should be created for the branch against the upstream repo

### What Happens instead

An error is shown in the extension that says "Resource not accessible by integration"

![image](https://user-images.githubusercontent.com/686963/106650081-1643c680-6558-11eb-8ae2-9e6b21c9727f.png)

**Comment by RMacfarlane** [comment]
@joshaber @matthewisabel It looks like the codespaces token is missing permissions for this - the call it's making is just to create the PR, it shouldn't be trying to fetch anything from the parent.

**Comment by joshaber** [comment]
That's true but since the token is scoped to only the one repository it won't allow access to the parent to create the PR. This is the opposite side of, e.g., https://github.com/github/codespaces/issues/1585.

**Comment by RMacfarlane** [comment]
@burkeholland Can you reopen this in the github/codepsaces repo? I'm not able to transfer it there

**Comment by burkeholland** [comment]
https://github.com/github/codespaces/issues/2243
